core-header-panel.html

```diff
@@ -139,12 +139,24 @@
        * @event scroll
        */
 
-      computed: {
-        dropShadowClass: 'calcDropShadowClass(atTop, mode, shadow)',
-        mainContainerClass: 'calcMainContainerClass(mode)'
-      },
+      properties: {
+
+        animateDuration: {
+          value: 200
+        },
+
+        atTop: {
+          value: true
```

Comment:
Yup. By omitting `type` I'm sort of making these "private". I could just assign them in `ready` too?